platform/machine/qemu/cluster.go

```diff
@@ -134,12 +134,14 @@ func (qc *Cluster) NewMachine(userdata *conf.UserData) (platform.Machine, error)
 			"qemu-system-x86_64",
 			"-machine", "accel=kvm",
 			"-cpu", "host",
+			"-m", "1024",
 		}
 	case "arm64-usr":
 		qmCmd = []string{
 			"qemu-system-aarch64",
 			"-machine", "virt",
 			"-cpu", "cortex-a57",
+			"-m", "2048",
```

Comment:
Just to double-check, this is necessary with images built with `./build_image prod`?  On amd64, dev images seem to need more than 1024 MB (https://github.com/coreos/bugs/issues/1594) but prod images don't.